apps/namadillo/src/types.d.ts

```diff
@@ -75,16 +79,23 @@ export type Validator = Unique & {
   imageUrl?: string;
 };
 
+export type UnbondingValidator = Omit<
+  | (IndexerUnbond & {
+      timeLeft: string;
+    })
+  | "validator"
+>;
+
+export type BondingValidator = Omit<IndexerBond | "validator">;
+
 export type MyValidator = {
-  stakingStatus: string;
   stakedAmount?: BigNumber;
   unbondedAmount?: BigNumber;
   withdrawableAmount?: BigNumber;
   validator: Validator;
-};
-
-export type MyUnbondingValidator = MyValidator & {
-  timeLeft: string;
+  bonds: BondingValidator[];
+  unbonding: UnbondingValidator[];
+  withdrawable: UnbondingValidator[];
```

Comment:
Maybe my naming confusion root issue is here, how a `MyValidator` could contain and array of `UnbondingValidator`? Should it be `UnbondingItems`?

Another question is if we should have different types for `unbonding` and `withdrawable`, like `UnbondingItem` and `WithdrawableItem`